ktor-client/ktor-client-core/common/src/io/ktor/client/call/SavedCall.kt

```diff
@@ -15,8 +15,41 @@ import io.ktor.utils.io.core.*
 import kotlinx.coroutines.*
 import kotlin.coroutines.*
 
+internal open class SavedHttpCall(client: HttpClient) : HttpClientCall(client) {
```

Comment:
Could you explain why we need the inheritance here?

Reply:
WIll remove `open`, thanks